flows/actions/send_broadcast.go

```diff
@@ -53,7 +53,7 @@ func (a *SendBroadcastAction) Execute(run flows.FlowRun, step flows.Step, log fl
 	}
 
 	translations := make(map[utils.Language]*events.BroadcastTranslation)
-	languages := append(utils.LanguageList{run.Flow().Language()}, run.Flow().Translations().Languages()...)
+	languages := append(utils.LanguageList{run.Flow().Language()}, run.Flow().Localization().Languages()...)
```

Comment:
in the flow spec we call the top level thing `localization` so I've renamed it in the code and that helps distinguish it from languageTranslations and itemTranslations which it's composed of.